core/jobs/src/test/java/com/cognifide/aet/job/common/modifiers/cookie/CookieModifierTest.java

```diff
@@ -71,7 +71,7 @@
   @Before
   public void setUp() throws Exception {
     when(wrapper.getWebDriver()).thenReturn(webDriver);
-    when(wrapper.getHttpRequestBuilder()).thenReturn(builder);
+    when(wrapper.getHttpRequestBuilder()).thenReturn(requestExecutor);
```

Comment:
What do you think about also changing the method name from `getHttpRequestBuilder()` to `getHttpRequestExecutor()` ?

Reply:
Thanks, I missed that.